examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<p>Let's change all future tenses to present. (At the time of this notebook getting into the main branch, all statements will have been already made true.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>